apps/opik-backend/src/main/java/com/comet/opik/api/resources/v1/priv/TraceResource.java

```diff
@@ -100,6 +101,16 @@ public Response getById(@PathParam("id") UUID id) {
                 .block();
     }
 
+    @GET
+    @Path("/workspaces-traces")
+    @Operation(operationId = "getTracesCntForWorkspaces", summary = "Get traces cnt on previous day for all available workspaces", description = "Get traces cnt on previous day for all available workspaces", responses = {
+            @ApiResponse(responseCode = "200", description = "TraceCntResponse resource", content = @Content(schema = @Schema(implementation = TraceCntResponse.class)))})
+    public Response getTracesCntForWorkspaces() {
```

Comment:
We discussed this offline, but let's make sure this endpoint isn't reachable from the Internet, you'll need help from devops team for that.

Reply:
For now it uses same `AuthFilter` as other endpoints, so I guess it's not reachable and valid apiKey is expected. Later as discussed this endpoint should be deployed in a different way, which should be discussed with devops.

Reply:
I agree, this should be out of the responsibility of this backend service. It should be enforced at another level (network and reverse proxy).